web/modules/custom/rdf_entity_provenance/rdf_entity_provenance.behat.inc

```diff
@@ -120,17 +148,30 @@ class RdfEntityProvenanceSubContext extends DrupalSubContextBase {
    *
    * @Then (the ):referral (entity )should have a related provenance activity
    */
-  public function getActivityForReferral(string $referral): RdfInterface {
-    $id = !UrlHelper::isValid($referral, TRUE) ? $this->getRdfEntityByLabel($referral)->id() : $referral;
-    /** @var \Drupal\rdf_entity_provenance\ProvenanceHelperInterface $helper */
-    $helper = \Drupal::service('rdf_entity_provenance.provenance_helper');
-    if (!$activity = $helper->loadProvenanceActivity($id)) {
+  public function assertActivityForReferral(string $referral): RdfInterface {
```

Comment:
As the we moved the logic of getting the provenance entity in a dedicated protected function, it doesn't make any sense that this assertion function to return something.